build-logic/src/main/kotlin/org.sonarsource.iac.code-style-convention.gradle.kts

```diff
@@ -14,7 +14,7 @@ spotless {
                 )
             )
         licenseHeaderFile(rootProject.file("LICENSE_HEADER")).updateYearWithLatest(true)
-        targetExclude("*/generated-sources/**")
+        targetExclude("*/generated-sources/**", "*/generated-src/**")
```

Comment:
Why not configure it to output to `generated-sources` as well?

Reply:
The `generated-src` was a default. We have to anyway configure it, so I'm ok to change it.